src/Hedgehog/Range.fs

```diff
@@ -77,7 +77,7 @@ module Range =
 
     /// Construct a range which is unaffected by the size parameter using the
     /// full range of a data type.
-    [<CompiledName("ConstantBounded")>]
+    [<CompiledName("--ConstantBounded")>]
```

Comment:
Is `--` some special trick that'll hide the member in other .NET languages? (I tried also `-` and it worked the same, I think.)

Reply:
One `-` would be the same yeah. It's just a way to set a name that's invalid for C# / VB.NET so you can't call it.

Reply:
Should we use just one `-` then? (The simplest possible thing that gets the job done.)

Reply:
Sure why not. Changed in dbc14b4a4bf18a790e084532796e07e56faece07

Reply:
:+1: :100: